### PR TITLE
added changelog for 1.3

### PR DESCRIPTION
**Pull request by unterstein**
Proposal for changelog 1.3 for current implementation state

-> Changelog for GPU support is done here https://github.com/mesosphere/marathon/pull/4180

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,7 +1,10 @@
 ## Changes from 1.2.0 to (unreleased version)
 
+## Changes from 1.1.0 to 1.3.0
```
Comment (meichstedt) [inline changelog.md]: Can you please move the `Changes from 1.2` below 1.3? The latest release version should be the top entry, as in

`1.1.0 to 1.3.0`
`1.2.0 skipped`
`1.0.0 to 1.1.0`

**Comment by unterstein** [comment]
retest this please

**Comment by unterstein** [comment]
retest this please

**Comment by unterstein** [comment]
retest this please

**Pull request merged**